packages/scandipwa/src/component/ProductAlerts/ProductAlerts.container.js

```diff
@@ -97,7 +97,7 @@ export class ProductAlertsContainer extends PureComponent {
         ).catch(
             /** @namespace Component/ProductAlerts/Container/ProductAlertsContainer/handlePriceDropSubscribe/then/catch */
             (error) => {
-                showErrorNotification('error', error[0].message);
+                showErrorNotification(error[0].message);
```

Comment:
1. Please refactor this in order to use `async / await` instead of Promise.
2. Please be careful here. Error is array only if it comes from graphql. If `catch` catches a regular javascript error, you will see strange notification and no error in console. This is hard to debug. Please reuse `getErrorMessage` from `Util/Request`.